libmpv/mpv.def

```diff
@@ -1,3 +1,5 @@
+__progname
```

Comment:
Note, this symbol is only inserted into applications. mpv build system appears to blindly use version script even if not necessary.